src/pages/pt-br/guides/deploy/aws.md

```diff
@@ -0,0 +1,234 @@
+---
+Título: Deploy seu Site Astro na AWS
+descrição: Como fazer o deploy do seu Astro Site na web usando AWS.
```

Comment:
Thanks for this @favanso!

We don’t need to translate the `title` and `description` keys here, only the text"

```suggestion
title: Deploy seu Site Astro na AWS
description: Como fazer o deploy do seu Astro Site na web usando AWS.
```